frontend/src/bundles/profile/store/slice.ts

```diff
@@ -33,9 +33,19 @@ const { reducer, actions, name } = createSlice({
             state.profile = action.payload;
             state.dataStatus = DataStatus.FULFILLED;
         });
-        builder.addMatcher(isAnyOf(disconnectSocialMedia.pending), (state) => {
-            state.dataStatus = DataStatus.PENDING;
+        builder.addCase(updateProfileAndEmail.fulfilled, (state, action) => {
```

Comment:
It looks like copy of code above. Consider using `addMatcher` and `isAnyOf`

Reply:
done